lightwood/api/json_ai.py

```diff
@@ -134,11 +134,10 @@ def lookup_encoder(
     # Time-series representations require more advanced flags
     if tss.is_timeseries:
         gby = tss.group_by if tss.group_by is not None else []
-        if col_name in tss.order_by + tss.historical_columns:
-            encoder_dict["module"] = "TimeSeriesEncoder"
-            encoder_dict["args"]["original_type"] = f'"{col_dtype}"'
-            encoder_dict["args"]["target"] = "self.target"
-            encoder_dict["args"]["grouped_by"] = f"{gby}"
+        if col_name in tss.order_by:
+            encoder_dict["module"] = "ArrayEncoder"
```

Comment:
Why not use the ArrayEncoder based timeseries encoder here ?

Reply:
Because `order_by` columns are timestamps that we assume have (mostly) regularly sampled data points. As such, in theory we won't gain much by computing moving averages over it.